### PR TITLE
A link to a more up to date framework :)

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+> **Info**: This project is **outdated** and **not maintained**. For an up to date framework
+with the latest and similar teck stack you should bookmark [**gae-init**](https://github.com/gae-init/gae-init).
```
Comment (kamalgill) [inline README.md]: @lipis: typo in "teck stack"... should be "tech stack" (or perhaps just use "stack")?

Comment (lipis) [inline README.md]: :+1:

**Comment by lipis** [comment]
@kamalgill Did you change your mind?

**Comment by kamalgill** [comment]
@lipis : Yes, for now at least, as PRs 77 and 78 has breathed some life into this project, and I hope to provide more updates in the near term.

**Comment by lipis** [comment]
sure man.. good luck!

**Comment by lipis** [comment]
Just a reality check.. is there any plan to actually update this project..? I really believe that is a bit unfair for the users to not mention that this project is outdated.. [commits](https://github.com/kamalgill/flask-appengine-template/commits/master), [releases](https://github.com/kamalgill/flask-appengine-template/releases)

**Comment by kamalgill** [comment]
@lipis: I just pushed an update to the readme with a link to your project.  Hope that helps!

**Comment by lipis** [comment]
@kamalgill Thanks man!

**Pull request closed without merging**